src/main/scala/gitbucket/core/util/Notifier.scala

```diff
@@ -19,8 +19,12 @@ import SystemSettingsService.Smtp
  * Please see the plugin for details.
  */
 trait Notifier {
+  def toNotify(subject: String, htmlMsg: String)
+    (recipients: Account => Session => Seq[String])(implicit context: Context): Unit = {
+    toNotify(subject, htmlMsg, None)(recipients)
+  }
 
-  def toNotify(subject: String, msg: String)
+  def toNotify(subject: String, htmlMsg: String, textMsg: Option[String])
```

Comment:
Well...either way, this fix affects other plugins...

@takezoe Well, you decide.